game/scripts/npc/abilities/talents/electrician_talent5_oaa.txt

```diff
@@ -3,23 +3,12 @@
   //=================================================================================================================
   // Electrician: Electric Shield bonus damage per second
   //=================================================================================================================
-  "special_bonus_electrician_electric_shield_damage"
+  "special_bonus_unique_electrician_electric_shield_damage"
```

Comment:
renaming this ability *actually* changes how it behaves. if you leave all my other changes but revert the rename then it doesn't actually work lolololololol (starting with `special_bonus_unique_` makes special things happen)